src/main/kotlin/com/github/biomejs/intellijbiome/settings/BiomeConfigurable.kt

```diff
@@ -0,0 +1,256 @@
+            row {
+                runFormatOnSaveCheckBox = checkBox(BiomeBundle.message("biome.run.format.on.save.label"))
+                    .bindSelected(RunOnObservableProperty(
+                        { settings.configurationMode != ConfigurationMode.DISABLED && settings.formatOnSave },
+                        { settings.formatOnSave = it },
+                        { !disabledConfiguration.isSelected && runFormatOnSaveCheckBox.isSelected }
+                    ))
+                    .component
+
+                val link = ActionsOnSaveConfigurable.createGoToActionsOnSavePageLink()
+                cell(link)
+            }.enabledIf(!disabledConfiguration.selected)
+
+            // *********************
+            // Apply safe fixes on save row
+            // *********************
+            row {
+                runSafeFixesOnSaveCheckBox = checkBox(BiomeBundle.message("biome.run.safe.fixes.on.save.label"))
+                    .bindSelected(RunOnObservableProperty(
+                        { settings.configurationMode != ConfigurationMode.DISABLED && settings.applySafeFixesOnSave },
+                        { settings.applySafeFixesOnSave = it },
+                        { !disabledConfiguration.isSelected && runSafeFixesOnSaveCheckBox.isSelected }
+                    ))
+                    .component
+
+                val link = ActionsOnSaveConfigurable.createGoToActionsOnSavePageLink()
+                cell(link)
+            }.enabledIf(!disabledConfiguration.selected)
+
+
+            // *********************
+            // Apply unsafe fixes on save row
+            // *********************
+            row {
+                runUnsafeFixesOnSaveCheckBox = checkBox(BiomeBundle.message("biome.run.unsafe.fixes.on.save.label"))
+                    .bindSelected(RunOnObservableProperty(
+                        { settings.configurationMode != ConfigurationMode.DISABLED && settings.applyUnsafeFixesOnSave },
+                        { settings.applyUnsafeFixesOnSave = it },
+                        { !disabledConfiguration.isSelected && runUnsafeFixesOnSaveCheckBox.isSelected }
+                    ))
+                    .component
+
+                val link = ActionsOnSaveConfigurable.createGoToActionsOnSavePageLink()
+                cell(link)
+            }.enabledIf(!disabledConfiguration.selected)
+
+            onApply {
+                biomeServerService.restartBiomeServer()
+                biomeServerService.notifyRestart()
+            }
+        }
+
+    }
+
+    private fun validateGlob(): ValidationInfoBuilder.(JBTextField) -> ValidationInfo? =
+        {
+            try {
+                FileSystems.getDefault().getPathMatcher("glob:" + it.text)
+                null
+            } catch (e: PatternSyntaxException) {
+                @NlsSafe val firstLine = e.localizedMessage?.lines()?.firstOrNull()
+                ValidationInfo(firstLine ?: BiomeBundle.message("biome.invalid.pattern"), it)
```

Comment:
If the pattern is incorrect, does Biome stop working?

Reply:
It's show a validation error

![CleanShot 2023-12-20 at 16 43 15@2x](https://github.com/biomejs/biome-intellij/assets/78874691/30a9713f-acd0-4bfa-8bbb-814941f2fde5)